File: src/BD.Business/Models/Item.cs
using System.Collections;
using System.Collections.Generic;

namespace BD.Business.Models
{
    public class Item : Entity
    {
        public string Name { get; set; }
        public int Quantity { get; set; }
        public decimal Price { get; set; }
        public string Description { get; set; }

        public IEnumerable<ReserveItem> ReserveItems { get; set; }
    }
}

File: src/BD.Frontend/src/app/core/models/item.ts
export class Item {
    id: number;
    name: string;
    quantity: number;
    price: number;
    description: string;
}

export class ItemSearch {
    id: number;
    name: string;
    minQuantity: number;
    maxQuantity: number;
    minPrice: number;
    maxPrice: number;
    enabled: boolean;
}
